app/manual/shared.html

```diff
@@ -1,35 +1,480 @@
+  <img src="img/sensitivity.png" />
+  <div class="caption">
+    Studying one-way sensitivity in deterministic analysis.
+  </div>
+</div>
+
+<h3 id="mcda-smaa">SMAA benefit-risk analysis</h3>
+
 <p>
-  Although nowadays withdrawn from the market due to the high incidence of severe depression and suicidal thoughts in
-  post-marketing surveillance studies, rimonabant was previously indicated as an adjunct to diet and exercise for the
-  weight management of overweight and obese patients. For illustrative purposes, we turn back time a few years and
-  consider the question of whether this medicinal product should also be granted marketing authorization for the
-  treatment of type 2 diabetes mellitus (DM2). The indication for which the benefit-risk assessment is conducted is the
-  use of rimobanant as an add-on therapy to either metformin or sulphonylurea monotherapy, and the patient population
-  consists of overweight or obese DM2 patients who remained inadequately controlled after having been treated with
-  either of these established first-line therapies for a period of at least 6 months. The alternatives under
-  consideration are rimonabant 5 mg/day, rimonabant 20 mg/day, and placebo, and the criteria based on which the
-  assessment is conducted are the change in fasting plasma glucose (favourable effect) and the incidence in
-  hypoglyceamia (unfavourable effect). The performance of the three treatments on these two criteria is summarized
-  here:
+  <a href="#mcda-smaa-theory">SMAA</a> explores the possibility space (that is,
+  the N-dimensional space with each alternative's measurements and and weights
+  as axes) defined by any uncertainty in both the data and the preferences for a
+  given problem and scenario. For example, when a user has used ranking
+  elicitation to define their preferences, and the data consists of point
+  estimates with normally-distributed confidence intervals, there is uncertainty
+  in both. SMAA will sample from these distributions while respecting any
+  constraints (ranking or otherwise) to create a combination of exact preference
+  weights and data values, and use the same process as on the
+  <a href="#mcda-deterministic-analysis">deterministic</a> page to calculate the
+  value of each alternative for this sampled situation. The set of values for
+  each sampled combination, when taken as a whole, can be analysed to show in
+  what fraction of the possibility space, for example, a given alternative
+  outperforms all others (i.e. has rank 1 in the rank acceptability plot). For
+  more detail, see the
+  <a href="#mcda-smaa-theory">theoretical section on SMAA</a>.
+</p>
+
+<div class="figure">
+  <img src="img/smaa1.png" />
+  <div class="caption">SMAA, ranking part.</div>
+</div>
+
+<div class="figure">
+  <img src="img/smaa2.png" />
+  <div class="caption">SMAA, central weights.</div>
+</div>
+
+<!-- theoretical section -->
+<h2 id="mcda-benefit-risk-analysis">Benefit-risk analysis</h2>
+<p>
```

Comment:
I would put a link to this in the suggested application summary at the top. Sth. like "For a theoretical background of the application, see this section".

Reply:
line 9